python/lsst/analysis/tools/atools/genericBuild.py

```diff
@@ -106,9 +116,139 @@ class FluxesDefaultConfig(Config):
     ref_matched = ConfigField[FluxConfig](doc="Reference catalog magnitude")
 
 
```

Comment:
Why are these things in genericBuild?

Reply:
The selectors have been moved to the selectors file.